uploader.html

```diff
@@ -18,73 +18,70 @@
       top: 15px;
       right: 15px;
     }
-    .select-label {
-      /*background-color: lightcoral;*/
+    #select-file-section {
+      padding-top: 15px;
+    }
+    .select-label, .current-label {
       text-align: right;
       font-size: 12px;
       font-weight: bold;
       padding-top: 7px;
     }
     .select-btn {
-      /*background-color: darkcyan;*/
+      padding-left: 10px;
     }
     .select-file {
       display: inline-block;
-      color: #ffffff;
-      background-color: #004ebc;
-      border: 1px solid #0043a3;
+      color: #fff;
+      background-color: #2e558c;
+      border: 1px solid #254674;
       border-radius: 3px;
       font-weight: normal;
       cursor: pointer;
-      /*background-color: mediumturquoise;*/
-    }
-    #current {
-      /*background-color: rebeccapurple;*/
-    }
-    #select-file-section {
     }
     #media-data-section {
-    }
-    .hide {
-      visibility: hidden;
-    }
-    .spacing {
-      padding-top: 40px;
+      padding-left: 5px;
     }
     .file-data,
-    .file-progress,
+    .progress-bar,
     .ready-text {
-      font-size: 0.85em;
+      font-size: 12px;
+      border-radius: 3px;
+      margin-top: 3px;
     }
     .file-data {
       display: inline-block;
-      background-color: green;
-      border: 1px solid black;
-      border-radius: 3px;
-      color: white;
       height: 40px;
+      background-color: #a3d8a3;
+      border: 1px solid #82c982;
+      color: white;
+      padding-left: 3px;
     }
-    progress[value] {
-      height: 30px;
+    .progress-bar {
+      width: 359px;
     }
     .ready-text {
-      color: green;
+      color: #82c982;
+      padding-left: 0;
     }
-    .js-current-label {
-      text-align: right;
-      font-size: 0.85em;
-      font-weight: bold;
-      padding: 12px 0;
-      /*background-color: darkkhaki;*/
-    }
-    .js-current-preview {
+    .current-preview {
       padding-bottom: 20px;
-      /*background-color: lightsalmon;*/
     }
     #js-output-file {
```

Comment:
it's generally better practice to give a separate styling class than the id referenced in javascript